pavelib/i18n.py

```diff
@@ -74,11 +71,13 @@ def i18n_dummy():
     Simulate international translation by generating dummy strings
     corresponding to source strings.
     """
-    cmd = "i18n_tool dummy"
-    sh(cmd)
+    sh("i18n_tool dummy")
     # Need to then compile the new dummy strings
-    cmd = "i18n_tool generate"
-    sh(cmd)
+    sh("i18n_tool generate")
+
+    # Generate static i18n JS files.
+    for system in ['lms', 'cms']:
+        sh(django_cmd(system, DEFAULT_SETTINGS, 'compilejsi18n'))
```

Comment:
@nedbat @dianakhuang @cahrens I just discovered that this has broken running ``paver i18n_dummy`` on sandboxes. I think this command now needs to take a settings command so that I can run it as ``paver i18n_dummy --settings=aws``. I guess nobody has been evaluating their i18n on sandboxes since this merged!